WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -75,4 +75,12 @@ def person_data
       dob: @person.dob,
     }
   end
+
+  def compute_auxiliary_data
+  end
+
+  def do_compute_auxiliary_data
+    ComputeAuxiliaryData.perform_later unless ComputeAuxiliaryData.in_progress?
```

Comment:
Ah two jobs can be run at the exact same time as well, gotcha. I think I'd worry about this if there were thousands of WRT members and result submissions daily =)